GetStream/stream-chat-android
# [CAS-1020] Update Push Notification's docs

https://stream-io.atlassian.net/browse/CAS-1020

### 🎯 Goal

Update Push Notification's docs

### 🛠 Implementation details

1. CMS docs updated
2. Docs added to Docusaurus
3. Updated KDoc

### 🧪 Testing

1. Check [CMS docs](https://getstream.io/chat/docs/android/push_android/?language=kotlin)
2. Check Docusaurus docs

### ☑️ Checklist


### 🎉 GIF
![giphy](https://user-images.githubusercontent.com/17440581/120326658-3fe3e300-c2e9-11eb-98a2-ee94d6a06b64.gif)
